parameters/example_population_params.json

```diff
@@ -2,7 +2,7 @@
     "disease":{
       "testParameters":{
           "pDiagnosticTestDetectsSuccessfully": 0.9,
-          "pDiagnosticTestAvailable":0.5
+          "pDiagnosticTestAvailableHour":0.5
```

Comment:
@paulbessell come back to this one. Not sure why it is Hour, but I should check the code

Reply:
Believe the logic (which might well be wrong!) is that there's a "pDiagnosticTestAvailableHour" chance that you get tested each *hour* after the symptomToTestingDelay (to model no availability at the testing centre etc). Maybe this should be a one shot thing (either you get the test exactly symptomToTestindDelay after symptoms, or you don't - for 0.5 anyway)?

Reply:
Interesting. My thoughts on this was that there would be a one-off probability that somebody would have a test available to them, rather than per hour. However, the testing behaviour will need a revamp when/if we come to require it in the model (which I think we will) but we would need some guidance on how to do this. Let's leave as is for now.

Reply:
Great - in that case let's leave the hour suffix for now since it respects the current behaviour.